### PR TITLE
adds client cert fields

### LINKED ISSUE #90: Support TLS client certificate and key per connection
**Is your feature request related to a problem? Please describe.**

The CLI will be adding support for client certificates in https://github.com/pomerium/cli/issues/35.

**Describe the solution you'd like**

The desktop UI should allow users to specify a cert and key for a given TCP connection.  

**Describe alternatives you've considered**

n/a

**Explain any additional use-cases**

n/a

**Additional context**

n/a

### PR DESCRIPTION
**Pull request by nhayfield**
## Summary
Adds UI for the client certs.
Requires CLI Bump 

## Related issues

Fixes #90 

## Checklist

- [x] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

File: src/renderer/pages/ConnectForm.tsx [hunk 1]
```diff
@@ -167,6 +210,48 @@ const ConnectForm: FC<Props> = () => {
...
+    setConnection((oldConnection) => {
+      set(oldConnection, 'clientCert.cert', new TextEncoder().encode(value));
+      return oldConnection;
+    });
+  };
+
+  const saveKeyText = (value: string): void => {
+    setKeyText(value);
+    setConnection((oldConnection) => {
+      set(oldConnection, 'clientCert.key', new TextEncoder().encode(value));
```
Comment (wasaga) [inline src/renderer/pages/ConnectForm.tsx:224]: why do we do it via `lodash` and not via standard setters provided?

**Review by wasaga — APPROVED** [review]
1. see some minor suggestions
2. requires https://github.com/pomerium/cli/pull/37 to work properly

File: src/renderer/pages/ConnectForm.tsx (hunk 1 above)
Comment (nhayfield) [reply src/renderer/pages/ConnectForm.tsx:224]: clientCert could be undefined because of the nested protobuf and lodash protects against that

Comment (nhayfield) [reply src/renderer/pages/ConnectForm.tsx:224]: also there are no standard setters with protos-ts it is just json.

**Comment by travisgroth** [comment]
We've cut cli v0.17 with the requisite changes https://github.com/pomerium/cli/releases/tag/v0.17.0

**Pull request merged**